tests/test_habitat_quality.py

```diff
@@ -1048,9 +1048,12 @@ def test_habitat_quality_case_insensitivty(self):
         habitat_quality.execute(args)
 
         # Reasonable to just check quality out in this case
+        #assert_array_sum(
+        #    os.path.join(args['workspace_dir'], 'quality_c.tif'),
+        #    5852.088)
```

Comment:
Was this left over from debugging, or should we still be asserting this?

Reply:
Huh, it must have been from debugging the new test results. It's a duplicate to an assertion right below it. I removed it. https://github.com/natcap/invest/pull/1159/commits/85a6d23583e268313d5838cea43b069a18882d60